ynr/apps/people/tests/test_person_models.py

```diff
@@ -64,6 +64,32 @@ def test_current_elections_standing_down(self):
             person.current_elections_standing_down(), [self.election]
         )
 
+    def test_current_elections_not_standing_then_standing_again(self):
```

Comment:
This is really testing that the standing down feature works, but it's not testing the web interface can change the `not_standing` list. 

Can you change this test to create a person who is not standing in an election, and then use the front end (e.g `self.app`) to post a new election, and then test that the election has been removed from the not_standing list?

Reply:
Fixed with 764ba75e2b1fd3f523da0a66ab7495b8f4260ce3